homeassistant/components/gios/strings.json

```diff
@@ -16,7 +16,7 @@
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
     },
     "abort": {
-      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%] for this measuring station"
```

Comment:
Should we use `already_configured_location` when that is available?

Reply:
IMO, `already_configured_location` doesn't fit here because this integration uses `station_id` and not station coordinates.

Reply:
Practically the id means location, right? It's just that we don't ask for coordinates in the form. But ok.

Reply:
Practically yes. I can change that if you think that we should use `already_configured_location` here.

Reply:
As codeowner you should decide what fits best for the integration.

Reply:
After thinking about it, I admit that `already_configured_location` will fit here. I'll change it.